requirements/edx/github.txt

```diff
@@ -57,6 +57,7 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 -e git+https://github.com/edx/edx-proctoring.git@release-2015-07-29#egg=edx-proctoring
+-e git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations
```

Comment:
@mattdrayer try:

    git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations==$setup.py_version

You can remove the '-e'

We're working on getting proper documentation to explain this.